src/mx_bluesky/I24/serial/fixed_target/i24ssx_moveonclick.py

```diff
@@ -17,10 +20,10 @@
 # Register clicks and move chip stages
 def onMouse(event, x, y, flags, param):
     if event == cv.EVENT_LBUTTONUP:
-        print("Clicked X and Y", x, y)
+        logger.info("Clicked X and Y %s %s" % x, y)
```

Comment:
```suggestion
        logger.info("Clicked X and Y %s %s" % (x, y))
```

pretty sure this is required, or at least used by convention. Please check throughout PR, no further comments on similar cases.

Reply:
Ah yes, definitely a typo, good spot.